spec/acceptance/enum_traits_spec.rb

```diff
@@ -3,6 +3,7 @@
     define_model("Task", status: :integer) do
       enum status: { queued: 0, started: 1, finished: 2 }
     end
+    Task.reset_column_information
```

Comment:
i had to add these, otherwise AR cached the column information and the tests fail (depending on the test execution order the integers are converted to strings or vice versa)